src/session.ts

```diff
@@ -377,6 +331,66 @@ export class SessionManager implements Middleware {
             return resolvedCodeLens;
     }
 
+    private async showExePathSettingDeprecationWarning(): Promise<void> {
+        const choice: string = await vscode.window.showWarningMessage(
+            "The 'powershell.developer.powerShellExePath' setting is deprecated, use " +
+            "'powershell.powerShellExePath' instead.",
+            "Fix Automatically");
+
+        if (!choice) {
+            return;
+        }
+
+        this.suppressRestartPrompt = true;
+        await Settings.change("powerShellExePath", this.sessionSettings.developer.powerShellExePath, true);
+        await Settings.change("developer,powerShellExePath", undefined, true);
```

Comment:
Is that suppose to be a comma?
```suggestion
        await Settings.change("developer.powerShellExePath", undefined, true);
```